avaje-prisms/src/main/java/io/avaje/prism/internal/ProcessorUtils.java

```diff
@@ -0,0 +1,196 @@
+package io.avaje.prism.internal;
+
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.annotation.processing.Generated;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.VariableElement;
+
+@Generated("avaje-prism-generator")
+final class ProcessorUtils {
```

Comment:
Should it be public?    We have public methods so in that sense we should have public here as well?  (or remove the public from the methods?)

Reply:
Oh, it actually is `public` when written I see ...